cmd/webhook_v2.go

```diff
@@ -40,7 +40,8 @@ func newWebhookV2Cmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "webhook",
 		Short: "Scaffold a webhook for an API resource.",
-		Long:  `Scaffold a webhook for an API resource. You can choose to scaffold defaulting, validating and (or) conversion webhooks.`,
+		Long: `Scaffold a webhook for an API resource. You can choose to scaffold defaulting,` +
+			`validating and (or) conversion webhooks.`,
```

Comment:
I think you are missing an space to indent this properly.

Also, I would use double quotes instead of back quotes if no special functionality given by backquote is needed.

Reply:
> I think you are missing a space to indent this properly.

The gofmt check indentation issues. 

> Also, I would use double quotes instead of back quotes if no special functionality given by backquote is needed.

The long ones require ``

> And I also think you are missing an space after the comma
👍 yes, it we can address :-)